packages/react-tabster/package.json

```
@@ -35,10 +35,10 @@
   "dependencies": {
     "@fluentui/make-styles": "^9.0.0-alpha.25",
     "@fluentui/react-make-styles": "^9.0.0-alpha.42",
-    "keyborg": "^0.7.0-alpha.1",
+    "keyborg": "^0.7.1",
     "@fluentui/react-shared-contexts": "^9.0.0-alpha.15",
     "@fluentui/react-utilities": "^9.0.0-alpha.29",
-    "tabster": "0.7.0",
+    "tabster": "0.7.3",
```

AFAIR we use no strict versions for package direct deps no?

---

You're right, this should be a caret 👍 This has been here for some time now

```suggestion
    "tabster": "^0.7.3",
```